src/main/java/seedu/dictionote/logic/parser/DeleteNoteCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.dictionote.logic.parser;
+
+import static seedu.dictionote.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.dictionote.commons.core.index.Index;
+import seedu.dictionote.logic.commands.DeleteContactCommand;
+import seedu.dictionote.logic.commands.DeleteNoteCommand;
+import seedu.dictionote.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteCommand object
```

Comment:
Shouldn't it be `new DeleteNoteCommand object` instead?
This issue appears in other parts of the code as well